internal/driver/glfw/window.go

```diff
@@ -724,13 +726,17 @@ func (w *window) mouseClicked(_ *glfw.Window, btn glfw.MouseButton, action glfw.
 		if action == glfw.Press {
 			w.queueEvent(func() { wid.MouseDown(mev) })
 		} else if action == glfw.Release {
-			dragged := w.mouseDragged.(interface{}).(fyne.CanvasObject)
-			_, tappableDrag := dragged.(desktop.Mouseable)
-			if dragged != nil && tappableDrag {
-				mev.Position = w.mousePos.Subtract(w.mouseDraggedOffset)
-				w.queueEvent(func() { dragged.(desktop.Mouseable).MouseUp(mev) })
-			} else {
+			if w.mouseDragged == nil {
 				w.queueEvent(func() { wid.MouseUp(mev) })
+			} else {
+				dragged := w.mouseDragged.(interface{}).(fyne.CanvasObject)
+				_, tapable := dragged.(desktop.Mouseable)
+				if dragged != nil && tapable {
+					mev.Position = w.mousePos.Subtract(w.mouseDraggedOffset)
+					w.queueEvent(func() { dragged.(desktop.Mouseable).MouseUp(mev) })
+				} else {
+					w.queueEvent(func() { wid.MouseUp(mev) })
+				}
```

Comment:
```suggestion
				if dragged, ok := w.mouseDragged.(desktop.Mouseable); ok {
					mev.Position = w.mousePos.Subtract(w.mouseDraggedOffset)
					w.queueEvent(func() { dragged.MouseUp(mev) })
				} else {
					w.queueEvent(func() { wid.MouseUp(mev) })
				}
```

Reply:
Thanks. For some reason I could not get it to work with that casting last night, no idea what I was doing wrong!